apis/management.cattle.io/v3/k8s_defaults.go

```diff
@@ -0,0 +1,59 @@
+package v3
+
+var (
+	K8sVersionToSystemImages = map[string]RKESystemImages{
```

Comment:
@moelsayed @galal-hussein Looks good, but lets think of other naming to better reflect the purpose. `K8sVersionToRKESystemImages` and `RancherK8sVersionToSystemImages` perhaps?